ldapauthenticator/ldapauthenticator.py

```diff
@@ -12,7 +12,7 @@
 
 
 class LDAPAuthenticator(Authenticator):
-    server_address = Unicode(
+    server_address = List(
```

Comment:
This is a breaking change, if you use [`traitlets.Union`](https://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.Union) you should be able to maintain backwards compatibility.